### DISCUSSION TITLE
Add `Content-Disposition` header to files without downloading the file

### DISCUSSION THREAD

**Discussion by mlbiche** [directus/directus#16800] [Feature Requests]
The Directus documentation [states](https://docs.directus.io/reference/files.html#downloading-a-file) that adding the `download` query param to a link to a file allows to directly download the file. It adds the [`Content-Disposition`](https://docs.directus.io/reference/files.html#downloading-a-file) header so the downloaded file has a proper name.

Is it possible to also set the `Content-Disposition` header even without setting the `download` attribute, with the value set to `inline; filename=...`?

This way, accessing the file link would allow the user to preview it and then download it with the proper filename if it needs to?

Cheers 🤙

**Comment by paescuj** [comment]
Thanks for the idea 👍 I've just submitted a pull request which addresses this: #16809

**Comment by rijkvanzanten** [comment]
This feature request has been accepted and is queued to be implemented! You can follow along with the progress here: https://github.com/directus/directus/issues/17078
